maplibre-adapter/src/main/java/com/amplifyframework/geo/maplibre/http/AWSRequestSignerInterceptor.kt

```diff
@@ -106,8 +108,8 @@ internal class AWSRequestSignerInterceptor(
         }
 
         val httpUrl = Url(
```

Comment:
Haven't looked at Kotlin sdk but what is the reason that the port is needed for both the scheme and the port val below.

Reply:
It port value is not set then it defaults to schema.port
```
public data class Url(
    public val scheme: Scheme,
    public val host: Host,
    public val port: Int = scheme.defaultPort,
    public val path: String = "",
    public val parameters: QueryParameters = QueryParameters.Empty,
    public val fragment: String? = null,
    public val userInfo: UserInfo? = null,
    public val forceQuery: Boolean = false,
    public val encodeParameters: Boolean = true,
)
```